core/server/master/src/main/java/alluxio/master/metastore/rocks/RocksStore.java

```diff
@@ -50,32 +50,38 @@
 public final class RocksStore implements Closeable {
   private static final Logger LOG = LoggerFactory.getLogger(RocksStore.class);
   public static final int ROCKS_OPEN_RETRY_TIMEOUT = 20 * Constants.SECOND_MS;
-
+  private final String mName;
   private final String mDbPath;
   private final String mDbCheckpointPath;
   private final Collection<ColumnFamilyDescriptor> mColumnFamilyDescriptors;
-  private final DBOptions mDbOpts;
+  private DBOptions mDbOpts;
 
   private RocksDB mDb;
   private Checkpoint mCheckpoint;
   // When we create the database, we must set these handles.
   private List<AtomicReference<ColumnFamilyHandle>> mColumnHandles;
 
   /**
+   * @param name a name to distinguish what store this is
    * @param dbPath a path for the rocks database
    * @param checkpointPath a path for taking database checkpoints
    * @param columnFamilyDescriptors columns to create within the rocks database
-   * @param dbOpts db options
    * @param columnHandles column handle references to populate
    */
-  public RocksStore(String dbPath, String checkpointPath,
-      Collection<ColumnFamilyDescriptor> columnFamilyDescriptors, DBOptions dbOpts,
+  public RocksStore(String name, String dbPath, String checkpointPath,
+      Collection<ColumnFamilyDescriptor> columnFamilyDescriptors,
       List<AtomicReference<ColumnFamilyHandle>> columnHandles) {
     Preconditions.checkState(columnFamilyDescriptors.size() == columnHandles.size());
+    mName = name;
     mDbPath = dbPath;
     mDbCheckpointPath = checkpointPath;
     mColumnFamilyDescriptors = columnFamilyDescriptors;
```

Comment:
shall we close all members in this list?

Reply:
I checked all objects from `org.rocksdb` and those objects I left alone do not inherit `RocksObject`, which means they are not `AutoCloseable`.

Discussed with @yuzhu today too, it's tricky to tell if an object is `AutoCloseable` and it's even harder to write UTs or detectors to ensure...